JiraReporter/Views/summary.cshtml

```diff
@@ -221,154 +232,67 @@ else
-                <span style="color:black; font-size:14px;">Unassigned</span>
-            </td>
-            <td colspan="1" style="padding:5px; vertical-align:top; text-align:left; border:1px solid #DDD">
-                @if (Model.Timing.InProgressUnassignedTasksSecondsLeft > 0)
-                {
-                    <span style="color:black; font-size:14px;">@Model.Timing.InProgressUnassignedTasksTimeLeftString</span>
-                }
-                else
-                {
-                    <span style="color:black; font-size:14px;"> - </span>
-                }
-            </td>
-            <td colspan="1" style="padding:5px; vertical-align:top; text-align:left; border:1px solid #DDD">
-                @if (Model.Timing.OpenUnassignedTasksSecondsLeft > 0)
-                {
-                    <span style="color:black; font-size:14px;">@Model.Timing.OpenUnassignedTasksTimeLeftString</span>
-                }
-                else
-                {
-                    <span style="color:black; font-size:14px;"> - </span>
-                }
-            </td>
-            <td colspan="1" style="padding:5px; vertical-align:top; text-align:left; border:1px solid #DDD">
-                @if (Model.Timing.UnassignedTasksSecondsLeft > 0)
-                {
-                    <span style="color:black; font-size:14px;">@Model.Timing.UnassignedTasksTimeLeftString</span>
-                }
-                else
-                {
-                    <span style="color:black; font-size:14px;"> - </span>
-                }
-            </td>
-
-        </tr>
+            @if (Model.Timing.UnassignedTasksSecondsLeft == 0)
+            {
+                display = "display:none !important";
+                visibility = "visibility:hidden";
+            }
+            <tr style="@visibility; @display">
+                <td style="width:75px; height:55px; max-width:75px; max-height:55px; border-right:1px solid grey; padding-right:15px; padding-bottom:10px; text-align:center; vertical-align:middle; color:black">
+                    <img src="cid:unassigned_48.png" />
+                    <span style="color:black; display:inline-block; padding-top:5px">Unassigned</span>
+                </td>
+                <td style="padding:0px; padding-bottom:10px; height:55px">
+                    <div style="width: @Model.ChartMaxWidthString; max-width:@Model.ChartMaxWidthString; height:14px; overflow:hidden; padding-bottom:2px">
+                        <div style="display:inline-block; background-color:#337AB7; width:@Model.UnassignedRemainingChartPixelWidthString; height:14px; float:left; margin:0"></div>
+                        <div style="width:45px; height:14px; line-height:14px; display:inline-block; font-size:12px; padding-left:5px; float:left;color:black">@Model.Timing.UnassignedTasksTimeLeftString</div>
+                    </div>
+                </td>
+            </tr>
+        </tbody>
```

Comment:
@Sebyd 
View-urile astea au inceput sa devina cam mari. Poate poti sa le imparti in mai multe view-uri partiale, si eventual sa refolosesti anumite view-uri. 

Cum ar fi un view pentru o linie dintr-un grafic. Ai nevoie sa stii doar lungimea in px si culoarea.